server/protocols/jmap/src/main/java/org/apache/james/jmap/mailet/TextCalendarBodyToAttachment.java

```diff
@@ -19,27 +19,40 @@
 
 package org.apache.james.jmap.mailet;
 
-import java.io.IOException;
+import java.io.InputStream;
 import javax.mail.MessagingException;
 import javax.mail.Multipart;
 import javax.mail.Part;
 import javax.mail.internet.MimeBodyPart;
 import javax.mail.internet.MimeMessage;
 import javax.mail.internet.MimeMultipart;
-import javax.mail.util.SharedByteArrayInputStream;
 
 import org.apache.mailet.Mail;
 import org.apache.mailet.MailetException;
 import org.apache.mailet.base.GenericMailet;
 
 import com.google.common.annotations.VisibleForTesting;
 
-public class ICALBodyTextToAttachment extends GenericMailet {
-    private static final String TEXT_MIME_TYPE = "text/calendar";
+/**
+ * This mailet convert Content-Type fo MimeMessage from text/calendar to mulitpart/mixed
+ *
+ * The BodyPart should be getting from content of text/calendar
+ * And because it's changed content-type so "Content-transfer-encoding" could be changed also based on rfc2045#section-6.4
+ * (relationship between content-type and content-transfer-encoding.
```

Comment:
sorry I don't understand this sentence at all

Reply:
I changed